tests/composition/test_control.py

```diff
@@ -904,6 +904,45 @@ def test_multilevel_control(self, mode, benchmark):
         assert np.allclose(res, [40])
         benchmark(oComp.run, [5], bin_execute=mode)
 
+    @pytest.mark.control
+    @pytest.mark.composition
+    @pytest.mark.parametrize("mode", ["Python",
+                                      pytest.param("LLVM", marks=pytest.mark.llvm),
+                                      pytest.param("LLVMExec", marks=pytest.mark.llvm),
+                                      pytest.param("LLVMRun", marks=pytest.mark.llvm),
+                                     ])
+    def test_recurrent_control(self, mode, benchmark):
```

Comment:
I'm surprised pytest doesn't complain about the missing `benchmark` mark. please drop the param, you're not suing benchmark anyway